pages/common/duplicity.md

```diff
@@ -0,0 +1,23 @@
+# duplicity
+
+> Creates incremental, compressed, encrypted, and versioned backups and optionally uploads them to a variety of backend services.
```

Comment:
Hmm that's strange. A `,` before an `and` just feels weird. I'm no grammar expert by any means. pinging @waldyrious if he has some time to give his expert comments.